code/game/objects/items/stacks/medical.dm

```diff
@@ -12,6 +12,7 @@
 	resistance_flags = FLAMMABLE
 	max_integrity = 40
 	novariants = FALSE
+	merge_type = /obj/item/stack/medical
```

Comment:
Shouldnt ointment and bruise packs need their merge type set too? This unit test seems kind of pointless if it forces abstract types to have their merge type set since it will pass on any subtype of this that doesn't have it properly set

Reply:
woops, fixed